app/src/main/res/layout/fragment_immunization_record_detail.xml

```diff
@@ -0,0 +1,143 @@
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:filterTouchesWhenObscured="true"
+    app:layoutDescription="@xml/fragment_immunization_details_scene"
+    tools:context="ca.bc.gov.bchealth.ui.healthrecord.vaccine.VaccineRecordDetailFragment">
+
+    <include
+        android:id="@+id/toolbar"
+        layout="@layout/layout_toolbar"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/card_view"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/space_4_x"
+        android:layout_marginTop="@dimen/space_4_x"
+        android:layout_marginEnd="@dimen/space_4_x"
+        app:cardCornerRadius="4dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/toolbar">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/rect_background"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/space_4_x">
+
+            <com.google.android.material.imageview.ShapeableImageView
+                android:id="@+id/iv_vaccine"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/space_4_x"
+                android:src="@drawable/ic_vaccine_dose_1"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="@id/tv_vaccine_name"
+                app:layout_constraintBottom_toBottomOf="@+id/tv_vaccine_name"/>
+
+            <com.google.android.material.textview.MaterialTextView
+                android:id="@+id/tv_vaccine_name"
+                style="?textAppearanceHeadline3"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/space_2_x"
+                android:layout_marginEnd="@dimen/space_4_x"
+                android:layout_marginTop="@dimen/space_4_x"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toEndOf="@id/iv_vaccine"
+                app:layout_constraintTop_toTopOf="parent"
+                android:textSize="15sp"
+                android:textColor="@color/status_green"
+                android:text="Second dose of hepatitis A vaccine" />
```

Comment:
use string resource file for the text.

Reply:
This is currently dummy text. Need to update after API and DB integration.

Reply:
Noted